scripts/enable-aks-monitor.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+WORKSPACE_ID=$1
+AKS_ID=$2
+
+az monitor diagnostic-settings create \
+    -n DiagLogAnalytics \
+    --resource ${AKS_ID} \
+    --workspace ${WORKSPACE_ID} \
+    --logs '[{"category":"kube-apiserver","Enabled":true},{"category":"kube-controller-manager","Enabled":true},{"category":"cluster-autoscaler","Enabled":true},{"category":"kube-scheduler","Enabled":true},{"category":"kube-audit","Enabled":true}]' \
```

Comment:
thoughts on extracting this to a separate file so we get syntax highlighting and the tests can pick up breakage of the json?